### PR TITLE
add keywords

### PR DESCRIPTION
**Pull request by MiguelPF**
## 🚪 Why?

Keywords are important for several reasons, especially in the context of npm packages or other software projects:

1. Searchability on npm and Other Platforms
Keywords help users find your package when they search for related terms.
They improve the discoverability of your package by aligning it with terms developers might use to solve specific problems.
2. Audience Targeting
Well-chosen keywords help target the right audience by highlighting the core functionality and use cases of your package.
This ensures your package attracts users who are specifically looking for its capabilities.
3. Improved SEO (Search Engine Optimization)
Keywords also enhance your package's visibility in search engine results outside npm (e.g., Google).
When users search for related topics or issues, your package has a better chance of appearing in results.
4. Highlight Features and Use Cases
They succinctly describe the main features or unique selling points of your package.
Developers scanning through npm search results can quickly understand what your package offers.
5. Contextual Categorization
Keywords categorize your package within a particular ecosystem (e.g., build tools, file watchers, caching).
This helps tools and platforms index it properly, making it easier for others to include it in curated lists or recommendations.
6. Community Engagement
Keywords reflecting current trends or popular tools (e.g., AWS S3, pipelines, automation) make your project appear relevant.
This can attract community contributors and users who are engaged in these areas.
Example:
If someone is looking for a Node.js package to handle artifact generation, they might search for terms like “file processing,” “build tool,” or “file watcher.” By including these terms as keywords, Shadowdog becomes more likely to appear in their search results.

In summary, keywords serve as a bridge between what your project offers and what potential users are looking for, making them an essential aspect of your npm package metadata.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,7 +7,33 @@
...
+  "remote-cache",
+  "AWS-S3",
+  "dependency-tree",
+  "git-integration",
+  "rake-commands",
+  "socket-communication",
+  "schema-validation",
+  "open-source",
+  "MIT-license"
+],
```
Comment (gtrias) [inline package.json:36]: weird indention

Comment (gtrias) [reply package.json:36]: let's add "turborepo" :smiling_imp:
